src/test/java/frc/robot/subsystems/DriveSubsystemTest.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems;
+
+import static org.mockito.Mockito.verify;
+
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class DriveSubsystemTest {
+
+    @Mock DifferentialDrive driveTrain;
+
+    @Test
+    public void testArcadeDrive() {
+        DriveSubsystem driveSubsystem = new DriveSubsystem(driveTrain);
+        driveSubsystem.drive(1, 0);
+        verify(driveTrain).arcadeDrive(1, 0);
```

Comment:
Take a look at this test and see if you can figure out what it's testing

Reply:
Does it make sure that the `drive` method on the DriveSubsystem calls `arcadeDrive` on drive train object?

Reply:
Basically yeah. It makes sure that the arcadeDrive method was called with the specific arguments. You can do a lot of different things with these verifies, like counting the number of times a method is called